### PR TITLE
Update/db seed with entrypoint

### PR DESCRIPTION
**Pull request by mgtgecko**
Before this update we used hibernate in our production code to auto-generate tables and initialized them using data.sql from our resource folder. This approach is nice and easy for development, but not suitable for production, since all tables would be dropped and recreated after a restart of the application on the server (let's say after a crash).
Now the database is initialized once at startup from the .sql file (schemas and sample data) in mysql/db/entrypoint and never touched or modified at application startup or shutdown.

### CODE REVIEW

**Comment by TuringRs** [comment]
I am not quite sure if the test for the repository implementation will work if we change the data for the test database.

File: mysql/db/entrypoint/foren_schema_and_sample_data.sql [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `forum`
+--
+
+DROP TABLE IF EXISTS `forum`;
```
Comment (niruc100) [inline mysql/db/entrypoint/foren_schema_and_sample_data.sql:22]: this document still drops tables on startup

**Review by niruc100 — APPROVED** [review]

**Review by niruc100 — CHANGES_REQUESTED** [review]
missclicked

File: mysql/db/entrypoint/foren_schema_and_sample_data.sql (hunk 1 above)
Comment (mgtgecko) [reply mysql/db/entrypoint/foren_schema_and_sample_data.sql:22]: But not at every restart of our application. Only once when the database container is started.

**Review by TuringRs — APPROVED** [review]
Is alright

File: mysql/db/entrypoint/foren_schema_and_sample_data.sql (hunk 1 above)
Comment (niruc100) [reply mysql/db/entrypoint/foren_schema_and_sample_data.sql:22]: ok

**Review by niruc100 — APPROVED** [review]

**Pull request merged**